src/llsq.jl

```diff
@@ -1,5 +1,9 @@
 using Distributed, DistributedArrays
-@everywhere using Distributed, DistributedArrays
```

Comment:
@wcwitt  - note in particular this change. If I don't do this, then I can't import it into another package. Maybe one needs to do this from the outside? E.g. `@everywhere using ACEfit`? To be explored ...